ui/src/main/java/com/wireguard/android/fragment/BaseFragment.kt

```diff
@@ -108,6 +109,18 @@ abstract class BaseFragment : Fragment(), OnSelectedTunnelChangedListener {
         }
     }
 
+    fun setShortcutState(view: CompoundButton, checked: Boolean) {
```

Comment:
Can that just be a generic View? We don't actually need any CompoundButton members, right?